tailwind.config.js

```diff
@@ -15,6 +15,13 @@ module.exports = {
     },
   },
   theme: {
+    screens: {
```

Comment:
Remove this lines from config. These are the defaults. No need to add them

Reply:
https://tailwindcss.com/docs/breakpoints

Reply:
I added these lines in the second commit because the verification link was breaking ...

I believe that the default did not work, but I can take it out to test it again.

Thanks